site/src/content/docs/commands/zarf_dev_deploy.md

```diff
@@ -23,8 +23,8 @@ zarf dev deploy [flags]
 ```
       --adopt-existing-resources           Adopts any pre-existing K8s resources into the Helm charts managed by Zarf. ONLY use when you have existing deployments you want Zarf to takeover.
       --components string                  Comma-separated list of components to deploy.  Adding this flag will skip the prompts for selected components.  Globbing component names with '*' and deselecting 'default' components with a leading '-' are also supported.
-      --create-set stringToString          Specify package variables to set on the command line (KEY=value) (default [])
-      --deploy-set stringToString          Specify deployment variables to set on the command line (KEY=value) (default [])
+      --create-set stringToString          [alias for --set-variables] Specify package variables to set on the command line (KEY=value) (default [])
+      --deploy-set stringToString          [alias for --set-variables] Specify deployment variables to set on the command line (KEY=value) (default [])
```

Comment:
This was an expected result from changing the other flags. Need to look more into how dev deploy is generated and see about `--create-set-variables` `--deploy-set-variables` and `--deploy-set-values`